reddit_gold/controllers.py

```diff
@@ -35,10 +35,10 @@
 @add_controller
 class GoldController(RedditController):
     def GET_about(self):
-        return GoldInfoPage(_("gold"), show_sidebar=False).render()
+        return GoldInfoPage(_("gold"), show_sidebar=False, page_classes=["gold-page-ga-tracking"]).render()
 
     def GET_partners(self):
-        return GoldPartnersPage(_("gold partners"), show_sidebar=False).render()
+        return GoldPartnersPage(_("gold partners"), show_sidebar=False, page_classes=["gold-page-ga-tracking"]).render()
```

Comment:
These lines are a little too long (79 characters according to the [style guide](https://github.com/reddit/styleguide/blob/master/python/pep-0008.rst)), so it'd be better broken out as:

```
return GoldPartnersPage(
    _("gold partners"),
    show_sidebar=False,
    page_classes=["gold-page-ga-tracking"],
).render()
```

(with trailing commas for easier diff)